README.md

```diff
@@ -90,6 +90,7 @@ If you know how to fix an issue, or want something to be added, consider opening
 - [Zeno-](https://github.com/Zeno-)
 - [indriApollo](https://github.com/indriApollo)
 - [Billy-S](https://github.com/Billy-S)
+- [luk3yx](https://github.com/luk3yx)
```

Comment:
I haven't added myself to the contributors list because the PR I submitted is trivial.

Reply:
I'll remove it, sorry.